tests/system/providers/google/cloud/cloud_sql/example_cloud_sql_query_mysql.py

```diff
@@ -42,8 +42,10 @@
 )
 from airflow.utils.trigger_rule import TriggerRule
 
+# mypy: disable-error-code="call-overload"
+
 if TYPE_CHECKING:
-    from airflow.settings import Session
+    pass
```

Comment:
If there's nothing here, should we just remove it?

Reply:
I think we could change a bit our ruff rules configuration for allow `TCH005` and maybe `TCH004`: https://docs.astral.sh/ruff/rules/#flake8-type-checking-tch

```toml
# pyproject.toml

extend-select = [
    ...
    "TCH",
]

extend-ignore = [
    ...
    "TCH003",
]
```

Reply:
Ah yeah. That was automatically added by one of the linters after some import removal. removed those.

Reply:
Yeah it was added by TCH001 and TCH002

Reply:
Yeah. As a follow up PR maybe :).. This one is already huge.

Reply:
Yeah! Sure. I though it might also reformat other parts and add additional changes in files from 0 to 9000 🤣